webapp/src/Service/ImportExportService.php

```diff
@@ -618,6 +618,7 @@ public function importGroupsJson(array $data, ?string &$message = null, ?array &
                 'visible' => !($group['hidden'] ?? false),
                 'sortorder' => @$group['sortorder'],
                 'color' => @$group['color'],
+                'allow_self_registration' => @$group['allow_self_registration'],
```

Comment:
Maybe
```suggestion
                'allow_self_registration' => $group['allow_self_registration'] ?? false,
```

is better here?